src/NuGet.Core/NuGet.Packaging/Signing/Signatures/SignatureVerifySettings.cs

```diff
@@ -31,6 +31,11 @@ public sealed class SignatureVerifySettings
         /// </summary>
         public bool ReportUnknownRevocation { get; }
 
+        /// <summary>
+        /// Indicates if a signing certificate's chain that chains to an untrusted root should be reported.
```

Comment:
Indicates if a signing certificate that chains to an untrusted root should be reported.